specification/e-referrals-service-api.yaml

```diff
@@ -3509,10 +3514,87 @@ paths:
           $ref: 'components/schemas/responses/cancelReferral/502Response.yaml'
         '504':
           $ref: 'components/schemas/responses/cancelReferral/504Response.yaml'   
+  /STU3/CommunicationRequest/{ubrn}/$ers.generateCRI:
+    post:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        
+        Use this endpoint to generate a PDF file that summarises an advice and guidance request including the conversation. 
+        This can only be be used for a UBRN which is an advice and guidance request that has *not* been converted to a referral.
+
+        ## Supported security patterns
+        - Healthcare worker, user-restricted access
+
+        ### Healthcare worker, user-restricted access
+        In order to use this endpoint you must be an authenticated e-RS user and use one of the following e-RS roles:
+          - `REFERRING_CLINICIAN`
+          - `REFERRING_CLINICIAN_ADMIN`
+          - `SERVICE_PROVIDER_CLINICIAN`
+          - `SERVICE_PROVIDER_CLINICIAN_ADMIN`
+
+        You need to use the `Unique Booking Reference Number` (UBRN) of of an advice and guidance request.
+
+        ## Important note
+        As noted for the [[HYPERLINK_A007]] endpoint that applies to referrals rather than advice and guidance requests, 
+        a referral pathway in e-RS can be made up of more than one UBRN. As such, it is important that all clinical information is obtained from across all the related 
+        UBRNs in the pathway as referenced in [[HYPERLINK_A005]].
+
+        An advice and guidance request can only ever exist as a single entity with no related UBRNs *unless* the advice and guidance request is converted to a referral. 
+        This can be done via endpoint [[HYPERLINK_A027]]. When the conversion is complete, the advice and guidance request becomes a 
+        referral (retaining the same UBRN) and continues its existence as such. Meaning that it may gain related referrals.
+
+        The advice and guidance conversation that took place before such a conversion still forms part of the clinical information. 
+        This conversation *does* get included in the PDF file returned by [[HYPERLINK_A007]] endpoint that can be retrieved for what is now a referral.
+
+        ## Use case
+
+        As an authenticated user
+
+        I need to retrieve a clinical information summary (in PDF format) associated with the advice and guidance request
```

Comment:
missing the full stop at end of 3555

Reply:
I've based the formatting of this on A007 which also doesn't have a full stop here